ansible/roles/common_installs/tasks/main.yml

```diff
@@ -58,6 +58,7 @@
     - libfreetype6-dev
     - gradle
     - ranger  # cmdline file browser
+    - libffi-dev  # Required for installing github3py
```

Comment:
true agreed. this package is pretty small ~300kb so it's not a huge deal, but is nice for managing complexity